WIP-CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 
 - The project now requires `helm-diff >= 3.1.2`. Remove the old one (via `rm -rf ~/.local/share/helm/plugins/helm-diff/`), before reinstalling dependencies.
 - Changed the way connectors are provided to dex
+- ck8s version validator check for the exact branch for the version number to match with the config.
```

Comment:
Maybe update to something like this, to make it clearer.
```suggestion
- CK8S version validation accepts version number if exactly at the release tag, otherwise commit hash of current commit.
  "any" can still be used to disable validation.
```